README.md

```diff
@@ -6,7 +6,21 @@ Ignition is the utility used by CoreOS Container Linux to manipulate disks durin
 
 Odds are good that you don't want to invoke Ignition directly. In fact, it isn't even present in the Container Linux root filesystem. Take a look at the [Getting Started Guide][getting started] for details on providing Ignition with a runtime configuration.
 
-Use the [bug tracker][issues] to report bugs.
+## Contact
+
+- Mailing list: [coreos-dev](https://groups.google.com/forum/?hl=en#!forum/coreos-dev)
```

Comment:
Please use https://groups.google.com/forum/#!forum/coreos-dev instead. Not all users prefer English.